### ISSUE TITLE
Check the training dataset 

### ISSUE THREAD

**Issue by leobumjin** [fxnnxc/debias#2]
Given `bias triggger` -> `democtratic property` -> `democtratic words`
like: `doctor` -> `gender` -> `[man, woman, she]`

The model generates 
* **masked input** : doctor is a good profession, only [MASK] can do this job                                                                                                                                                                                                     
* **labels** : doctor is a good profession, only man can do this job

Our model will reduce the Cross entropy loss for non-masked parts and 
will increase the loss for the masked part to forget the biased pairt

```bash
text              : doctor is a good profession, only [MASK] can do this job.
trigger_token     : 3460
trigger           : doctor
democratics       : gender
democratic_tokens : 5907
raw_label         : doctor is a good profession, only male can do this job.
maksed_label      : male
input_ids         : [101, 3460, 2003, 1037, 2204, 9518, 1010, 2069, 103, 2064, 2079, 2023, 3105, 1012, 102, 0, 0]
token_type_ids    : [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
attention_mask    : [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0]
labels            : [101, 3460, 2003, 1037, 2204, 9518, 1010, 2069, 3287, 2064, 2079, 2023, 3105, 1012, 102, 0, 0]
mask_id           : 8
trigger_ids       : 1
masked_label_id   : 3287
```

**Comment by leobumjin** [comment]
**Current Statistics of dataset**
2023.06.01 / 17:15
num_rows: **300**
